src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/StandardJourneyStepDefs.scala

```diff
@@ -162,9 +162,9 @@
 //    CommodityCodeDetailsPage.checkPageTitle()
```

Comment:
Why these are commented?

Reply:
This is old stepdef were you have written all the steps in one step def and the entire file is commented out. not sure why this is showing now.

Reply:
hmm..